docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -23,21 +25,95 @@ If the tested page does not use HTTPS-specific features (like
 [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API),
 [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API),
 [ApplePaySession](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession), or
-[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)), TestCafe successfully proxies it over HTTP. Otherwise, the tests fail because the page throws JavaScript errors, does not load or loads partially. This indicates that you should [configure the proxy server to use HTTPS](#proxy-the-website-over-https).
+[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)), TestCafe successfully proxies it over HTTP. Otherwise, the tests fail because the page throws JavaScript errors, does not load or loads partially.
 
-### Proxy the Website Over HTTPS
+### Use a Self-Signed Certificate
+
+To establish a secure HTTPS-connection, Node.js requires a valid [PKCS12](https://en.wikipedia.org/wiki/PKCS_12) certificate (`.pfx`). To launch tests from the command line, you need to [generate one manually](#generate-a-certificate-from-the-command-line). In API, use an [automatic module](#generate-a-certificate-in-your-code), unless you are willing to [use a trusted certificate](#use-a-trusted-certificate).
+
+#### Generate a Certificate From the Command Line
+
+In **bash**, a certificate chain can be generated using [OpenSSL](https://www.openssl.org/docs/man1.1.1/man1/openssl-req.html).
```

Comment:
> using

We can remove this '-ing' and passive voice if we rephrase

`In **bash**, you can use [OpenSSL](https://www.openssl.org/docs/man1.1.1/man1/openssl-req.html) to generate a certificate chain.`